models/category/src/types.ts

```diff
@@ -22,6 +22,6 @@ export type TCategoryGraphql = Omit<
   __typename: 'Category';
   createdBy: TClientLoggingGraphql;
   lastModifiedBy: TClientLoggingGraphql;
-  nameAllLocales?: TLocalizedStringGraphql;
-  descriptionAllLocales?: TLocalizedStringGraphql;
+  nameAllLocales: TLocalizedStringGraphql | null;
+  descriptionAllLocales: TLocalizedStringGraphql | null;
```

Comment:
Looking at the generated types, optional fields should still get the `?` (and additionally the `null`)

https://github.com/commercetools/merchant-center-application-kit/blob/7dabb2768017de78a47a9a6c4673922834aabe12/packages/application-shell/src/types/generated/ctp.ts#L2073

```suggestion
  nameAllLocales?: TLocalizedStringGraphql | null;
  descriptionAllLocales?: TLocalizedStringGraphql | null;
```